spec/features/visitor_can_login_spec.rb

```diff
@@ -4,29 +4,36 @@
   scenario 'and is redirected to the users dashboard' do
     user = create(:user)
 
-    visit farmersmarket_path
-    click_on "Login"
+    visit farmers_markets_path
```

Comment:
I made a note about making this snake case on someone else's pull request, I like it!